content/articles/2018-07-broom-0-5-0.markdown

```diff
@@ -0,0 +1,169 @@
+---
+title: broom 0.5.0
+author: Alex Hayes
+date: "2018-07-18"
+slug: broom-0-5-0
+description: > 
+  broom 0.5.0 is on CRAN.
+categories:
+  - package
+photo:
+  url: https://unsplash.com/photos/vYcH7pI6v1Q
+  author: Nagesh Badu
+---
+
+
+
+<html>
+<style>
+h2 code {
+    font-size: 1em;
+    
+}
+</style>
+</html>
+
+We are delighted to announced that [broom](https://broom.tidyverse.org/) 0.5.0 is now available on CRAN. broom 0.5.0 is a major new release featuring changes that affect both users and developers. See the [News]( https://broom.tidyverse.org/news/) for a detailed list of changes.
+
+## Tibble output
+
+All tidiers should now return `tibble`s rather than `data.frame`s. This allows broom to take advantage of the nice tibble print method and the more consistent behavior of tibbles:
+
+
+```r
+library(broom)
+
+fit <- lm(mpg ~ ., mtcars)
+tidy(fit)
+```
+
+```
+## # A tibble: 11 x 5
+##   term        estimate std.error statistic p.value
+##   <chr>          <dbl>     <dbl>     <dbl>   <dbl>
+## 1 (Intercept)  12.3      18.7        0.657   0.518
+## 2 cyl          -0.111     1.05      -0.107   0.916
+## 3 disp          0.0133    0.0179     0.747   0.463
+## 4 hp           -0.0215    0.0218    -0.987   0.335
+## 5 drat          0.787     1.64       0.481   0.635
+## # ... with 6 more rows
+```
+
+
+These changes will mostly likely affect you when you:
+
+- subset with `[`, which always returns a tibble.
+- set rownames on a tibble, which is deprecated.
+- use augment methods on models with matrix covariates specified in a formula, which will error.
+
+`augment()` will error with matrix covariates because tibbles are more strict about their contents than data frames. Data frames allow users to specify columns in a matrix, like so:
```

Comment:
I think this section all the way down to (but not including) "Support for matrix columns..." needs to be either removed from the blog post or moved into an appendix; otherwise having these details here will distract from the other top-level broom changes